src/hooks/useWorldBySlug.ts

```diff
@@ -0,0 +1,68 @@
+import { useFirestore, useFirestoreCollectionData } from "reactfire";
```

Comment:
Maybe it's best to use a single `useWorld()` hook that can distinguish search by `id` or `slug`.